### ISSUE TITLE
Problem using GPU

### ISSUE THREAD

**Issue by koopa31** [kevinjohncutler/omnipose#38]
Hi,
I've been trying to use Omnipose with GPU for two days. Actually, Cellpose works like charm, but when I try to use Omnipose I always get this error: RuntimeError: cuDNN error: CUDNN_STATUS_NOT_INITIALIZED.
When I do torch.cuda.isavailable() I get "True", so I don't understand. I tried to remove and re install torch with cuda, several times with different versions, but I always get the same error. Could you help me please?
Thank you.

**Comment by koopa31** [comment]
Actually, I succeeded in making it work on Windows, but I keep getting the same problem on Ubuntu 22.04. Could that be related to my Linux version?

**Comment by kevinjohncutler** [comment]
@koopa31 Yes, it must be something to do with your system, drivers, and torch version. What is your GPU and cuda/driver info? The only thing I can think of that might allow Cellpose to work but not Omnipose is that I could have made the install dependencies for Omnipose too new to work on your system. Some older graphics cards will not work with newer versions of pytorch (torch needs more recent drivers). A quick check is to compare the `conda list` output in your cellpose and omnipose environments.
